server/cluster/metadata/topology_manager.go

```diff
@@ -176,6 +178,21 @@ func (m *TopologyManagerImpl) GetTableIDs(shardIDs []storage.ShardID) map[storag
 	return shardTableIDs
 }
 
+func (m *TopologyManagerImpl) GetShardIDs(tableIDs []storage.TableID) map[storage.TableID]storage.ShardID {
+	m.lock.RLock()
+	defer m.lock.RUnlock()
+
+	tableShardMapping := make(map[storage.TableID]storage.ShardID, len(tableIDs))
+	for _, tableID := range tableIDs {
+		shardID, exists := m.tableShardMapping[tableID]
+		if exists {
+			tableShardMapping[tableID] = shardID
+		}
```

Comment:
Return error if tableID not exists?

Reply:
`TopologyManager.GetShardIDs` receive a `TableID` slice as parameter, if return error when some `tableID` not found in any shard, it means this function execute failed and we could not use it's execution result, this will make it impossible for us to judge which `TableID` exist on shards, I think the current implementation is a bit better.